<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>WishlistItem</class>
 <widget class="QWidget" name="WishlistItem">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>60</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>0</width>
    <height>60</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>16777215</width>
    <height>60</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background: rgb(237, 237, 237);</string>
  </property>
  <layout class="QHBoxLayout" name="layout">
   <property name="spacing">
    <number>0</number>
   </property>
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <item>
     <widget class="QLabel" name="coverLabel">
       <property name="sizePolicy">
         <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
         </sizepolicy>
       </property>
       <property name="minimumSize">
         <size>
           <width>100</width>
           <height>60</height>
         </size>
       </property>
       <property name="maximumSize">
         <size>
           <width>100</width>
           <height>60</height>
         </size>
       </property>
       <property name="text">
         <string/>
       </property>
     </widget>
   </item>
   <item>
     <widget class="QWidget" name="root">
       <layout class="QHBoxLayout" name="rootLayout" stretch="1,0,0,0,0,0">
        <property name="spacing">
          <number>10</number>
        </property>
        <property name="leftMargin">
          <number>10</number>
        </property>
        <property name="topMargin">
          <number>0</number>
        </property>
        <property name="rightMargin">
          <number>10</number>
        </property>
        <property name="bottomMargin">
          <number>0</number>
        </property>
        <item>
          <widget class="QLabel" name="titleLabel">
          <property name="font">
            <font>
            <pointsize>12</pointsize>
            <bold>true</bold>
            </font>
          </property>
          <property name="text">
            <string notr="true">Title</string>
          </property>
          </widget>
        </item>
        <item>
          <widget class="QWidget" name="rating" native="true">
          <layout class="QHBoxLayout" name="ratingLayout">
            <property name="spacing">
            <number>8</number>
            </property>
            <property name="leftMargin">
            <number>0</number>
            </property>
            <property name="topMargin">
            <number>0</number>
            </property>
            <property name="rightMargin">
            <number>0</number>
            </property>
            <property name="bottomMargin">
            <number>0</number>
            </property>
          </layout>
          </widget>
        </item>
        <item>
          <widget class="QWidget" name="platforms" native="true">
          <layout class="QHBoxLayout" name="platformsLayout">
            <property name="spacing">
            <number>8</number>
            </property>
            <property name="leftMargin">
            <number>0</number>
            </property>
            <property name="topMargin">
            <number>0</number>
            </property>
            <property name="rightMargin">
            <number>0</number>
            </property>
            <property name="bottomMargin">
            <number>0</number>
            </property>
            <item>
            <widget class="QLabel" name="windowsIcon">
              <property name="sizePolicy">
              <sizepolicy hsizetype="Fixed" vsizetype="Preferred">
                <horstretch>0</horstretch>
                <verstretch>0</verstretch>
              </sizepolicy>
              </property>
              <property name="minimumSize">
              <size>
                <width>16</width>
                <height>0</height>
              </size>
              </property>
              <property name="styleSheet">
              <string notr="true">image: url(:/icons/windows.svg);</string>
              </property>
              <property name="text">
              <string/>
              </property>
              <property name="alignment">
              <set>Qt::AlignmentFlag::AlignCenter</set>
              </property>
            </widget>
            </item>
            <item>
            <widget class="QLabel" name="macIcon">
              <property name="sizePolicy">
              <sizepolicy hsizetype="Fixed" vsizetype="Preferred">
                <horstretch>0</horstretch>
                <verstretch>0</verstretch>
              </sizepolicy>
              </property>
              <property name="minimumSize">
              <size>
                <width>16</width>
                <height>0</height>
              </size>
              </property>
              <property name="styleSheet">
              <string notr="true">image: url(:/icons/apple.svg);</string>
              </property>
              <property name="text">
              <string/>
              </property>
              <property name="alignment">
              <set>Qt::AlignmentFlag::AlignCenter</set>
              </property>
            </widget>
            </item>
            <item>
            <widget class="QLabel" name="linuxIcon">
              <property name="sizePolicy">
              <sizepolicy hsizetype="Fixed" vsizetype="Preferred">
                <horstretch>0</horstretch>
                <verstretch>0</verstretch>
              </sizepolicy>
              </property>
              <property name="minimumSize">
              <size>
                <width>16</width>
                <height>0</height>
              </size>
              </property>
              <property name="styleSheet">
              <string notr="true">image: url(:/icons/linux.svg);</string>
              </property>
              <property name="text">
              <string/>
              </property>
              <property name="alignment">
              <set>Qt::AlignmentFlag::AlignCenter</set>
              </property>
            </widget>
            </item>
          </layout>
          </widget>
        </item>
        <item>
          <widget class="QLabel" name="genreLabel">
          <property name="sizePolicy">
            <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
            </sizepolicy>
          </property>
          <property name="font">
            <font>
            <pointsize>10</pointsize>
            </font>
          </property>
          <property name="styleSheet">
            <string notr="true">color: gray;</string>
          </property>
          <property name="text">
            <string notr="true">Genre</string>
          </property>
          </widget>
        </item>
        <item>
          <widget class="QPushButton" name="removeButton">
          <property name="sizePolicy">
            <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
            </sizepolicy>
          </property>
          <property name="cursor">
            <cursorShape>PointingHandCursor</cursorShape>
          </property>
          <property name="text">
            <string/>
          </property>
          <property name="icon">
            <iconset resource="../resources.qrc">
            <normaloff>:/icons/xmark.svg</normaloff>:/icons/xmark.svg</iconset>
          </property>
          </widget>
        </item>
        <item>
          <widget class="QPushButton" name="buyButton">
          <property name="sizePolicy">
            <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
            </sizepolicy>
          </property>
          <property name="font">
            <font>
            <pointsize>9</pointsize>
            </font>
          </property>
          <property name="cursor">
            <cursorShape>PointingHandCursor</cursorShape>
          </property>
          <property name="styleSheet">
            <string notr="true">QPushButton
      {
          color: gray;
      }

      QPushButton:hover
      {
          background-color: qlineargradient(spread:pad, x1:0, y1:0, x2:1, y2:1, stop:0 rgb(180, 230, 60), stop:1 rgb(140, 202, 39));
          color: white;
      }</string>
          </property>
          <property name="text">
            <string notr="true">TBA</string>
          </property>
          </widget>
        </item>
      </layout>
     </widget>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="../resources.qrc"/>
 </resources>
 <connections/>
</ui>
